reactnative/README.md

```diff
@@ -0,0 +1,31 @@
+#React Native Implementation
+
+Visit the [React Native page](http://propertycross.com/reactnative/) on the PropertyCross website for screenshots.
+
+##Introduction
+
+[React Native](https://facebook.github.io/react-native/) enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and React. The focus of React Native is on developer efficiency across all the platforms you care about — learn once, write anywhere. Facebook uses React Native in multiple production apps and will continue investing in React Native.
```

Comment:
With the introduction section we try to avoid 'spin' ... 

> "enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and React"

This is clearly the way ReactNative would describe their framework ;-) Our preference is to make the introduction factual.